index.js

```diff
@@ -175,6 +184,19 @@ alexa.response = function(session) {
   };
 };
 
+alexa.directives = function(directives) {
+  // load the alexa response directives information into details
+  this.details = directives;
+
+  this.set = function(directive) {
+    this.details.push(directive);
+  };
+
+  this.clear = function() {
```

Comment:
@ajcrites This guy needs to specs. Mocha is wholly different than RSpec, so a little help here would be great when you get a chance!